src/partials/program__section.html

```diff
@@ -0,0 +1,64 @@
+<section class="program">
```

Comment:
https://www.loom.com/share/7242682ba69744668083fd7352a97c73

Reply:
Finish with this.